server/src/main/java/io/crate/types/DataTypeXContentExtension.java

```diff
@@ -63,6 +63,13 @@
                     b.value(row.get(i));
                 }
                 b.endArray();
+            }),
+            Map.entry(TimeTZ.class, (b, v) -> {
+                TimeTZ timetz = (TimeTZ) v;
+                b.startArray();
+                b.value(timetz.getMicrosFromMidnight());
+                b.value(timetz.getSecondsFromUTC());
+                b.endArray();
```

Comment:
I'm not sure if this is the best representation for HTTP.

@seut Do you have any inputs here how we should represent `time with time zone` to users via HTTP?

Reply:
I was not sure either, but as it makes the micros offset, as well as the seconds offset, explicit, I went with it. Please let me know your thoughts.

Reply:
I guess it makes sense to keep it in a format that is easy to parse for clients - we do the same for timestamp (long instead of text format)

We also have to extend the documentation here:

https://github.com/crate/crate/blob/e09e00db18273d070083b6dc4b659452c0eaee36/docs/interfaces/http.rst#L233-L237

and we should probably also follow up:
-  Add the time to the python client (mapping it to `datetime.time`) 
-  interpreting it in the Admin UI. (To have a human readable display as well)

cc @autophagy